plugins/woocommerce-admin/client/tasks/fills/experimental-products/card-layout.tsx

```diff
@@ -41,7 +41,7 @@ const CardLayout: React.FC< CardProps > = ( { items } ) => {
 										method: 'manually',
 									} );
 									recordCompletionTime();
-									window.location = getAdminLink(
+									window.location.href = getAdminLink(
```

Comment:
I updated the [/typings/index.d.ts](https://github.com/woocommerce/woocommerce/pull/34211/files#diff-4642695243afdb91bce1cf4782a7428a1e31e6c5e6d20f73c8196a169dbdaf49) file with the correct `window.location` type, which is why I updated a bunch of places to use `href` instead.

Reply:
Do we also need to add the `href` [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/wp-admin-scripts/onboarding-homepage-notice/index.js#L85) and [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/tasks/fills/products/product-template-modal.js#L91)?

Reply:
Yeah probably although I do believe it will work the way it is (weirdly enough). I missed these as they were not `TS` files so the TypeScript didn't complain, but will update those. Thanks for catching that.

Reply:
Done as part of: https://github.com/woocommerce/woocommerce/pull/34211/commits/dedd1a6e021c8d0f47b3bd68a3c4fb99a8f82d54